src/LibraryManager.Vsix/UI/Models/LibraryIdViewModel.cs

```diff
@@ -23,7 +23,15 @@ public LibraryIdViewModel(ISearchService packageSearchService, string initialSea
         protected override Task<CompletionSet> FilterCompletions(CompletionSet input)
         {
             CompletionSet result = input;
-            int atIndex = SearchText.IndexOf('@');
+            int atIndex = SearchText.LastIndexOf('@');
```

Comment:
If we wanted to simplify this a bit by making *a large and scary assumption*, we could just assume that an `@` at the beginning is for a scoped package, and there should only ever be a single `@` after that.  If we go on this assumption, we could just do `int atIndex = SearchText.LastIndexOf('@', 1);` and then skip the if block.

This is a technically correct (if inflexible - but it is simple!) option for all valid scenarios today.  There are currently no  scenarios (AFAIK?) that allow @ anywhere in the name except at the beginning for a scoped package.  If you have multiple after that, there isn't a strictly defined behavior.